packages/server/models/Theme.ts

```diff
@@ -0,0 +1,30 @@
+import { Column, DataType, Model, Table } from 'sequelize-typescript'
+
+@Table({
+  timestamps: false,
+  paranoid: true,
+  tableName: 'theme',
+})
+export class Theme extends Model<Theme> {
+  @Column({
+    type: DataType.INTEGER,
+    field: 'id_theme',
```

Comment:
[Предложение]
В 11 строке `id_theme`, а в 19 - `user_id`. Может в одном порядке слова сделать? 
```suggestion
    field: 'theme_id',
```

Reply:
fix